src/definitions/modules/slider.js

```diff
@@ -861,11 +861,12 @@ $.fn.slider = function(parameters) {
         },
 
         set: {
-          value: function(newValue) {
+          value: function(newValue, fireChange) {
+            fireChange = fireChange !== false;
             var toReset = previousValue === undefined;
             previousValue = previousValue === undefined ? module.get.value() : previousValue;
             module.update.value(newValue, function(value, thumbVal, secondThumbVal) {
-              if (!initialLoad || settings.fireOnInit){
+              if ((!initialLoad || settings.fireOnInit) && fireChange){
```

Comment:
> The change is backward compatible, because when the new parameter is omitted, it behaves like before (triggering the callback)

Hmm, it seems breaking change for me.
IIUC, if `fireChange` is not supplied, it will be `undefined`, which is `false`-y.
So this `if` block are skipped, therefore `onChange` callback are not invoked.

Reply:
@exoego fireChange is fetched at the start of the function by (same logic is inside calendar)
```javascript
            fireChange = fireChange !== false;
```

So, if the `fireChange` is not given (undefined) it's converted to `true` automatically to preserve the current logic
![image](https://user-images.githubusercontent.com/18379884/98206156-bbd7a180-1f39-11eb-8466-9314dd33d75b.png)

Reply:
Ah, I see.